### PR TITLE
Commands for managing public IP address reservations

### PR DESCRIPTION
**Pull request by bear454**

### CODE REVIEW

File: azurectl/app.py [hunk 1]
```diff
@@ -23,6 +24,6 @@ def __init__(self):
         app = CliTask(should_load_config=False)
         action = app.cli.get_command()
         service = app.cli.get_servicename()
-        task_class_name = service.title() + action.title() + 'Task'
+        task_class_name = camelize(service) + camelize(action) + 'Task'
```
Comment (schaefi) [inline azurectl/app.py]: that's nice :+1:

**Comment by schaefi** [comment]
I'm impressed, that's great work :smile:

merge it :+1:

File: azurectl/app.py (hunk 1 above)
Comment (bear454) [inline azurectl/app.py]: Except I need to package that dependency :/

Comment (schaefi) [inline azurectl/app.py]: >    Except I need to package that dependency :/

Oh this is not core python ? In this case I think we should stick with
the old code

Comment (bear454) [inline azurectl/app.py]: I'll write a local version of camelize instead. We really need something more robust than title... as using just title, we end up with ComputeReserved_IpTask .

**Comment by bear454** [comment]
If you don't mind; I'm going to refactor 'reserved_ip' to 'reserved-ip' in the commands, to be more consistent.

**Comment by schaefi** [comment]
looks all good to me, thanks for not introducing a new package dependency

**Pull request merged**